package/yast2-storage-ng.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Fri Mar 13 14:02:15 UTC 2020 - Steffen Winterfeldt <snwint@suse.com>
+
+- write nocow attribute correctly into YAML
```

Comment:
Miss the bug number

Reply:
There's no bug number. I wouldnät mind staying this in some queue som time...